filters/resize.go

```diff
@@ -78,13 +78,13 @@ func (r *resize) CreateFilter(args []interface{}) (filters.Filter, error) {
 	}
 
 	if len(args) == 3 {
-		f.proportion, err = parse.EskipBoolArg(args[2])
+		f.keepAspectRatio, err = parse.EskipBoolArg(args[2])
 
 		if err != nil {
 			return nil, err
 		}
 	} else {
-		f.proportion = true
+		f.keepAspectRatio = true
```

Comment:
Why do you think it should default to `true`? We have the `longerEdgeResize` filter which keeps the aspect ratio be default.